crates/rust-analyzer/src/config.rs

```diff
@@ -709,8 +709,10 @@ enum ManifestOrProjectJson {
 #[serde(rename_all = "snake_case")]
 enum MergeBehaviorDef {
     None,
-    Full,
-    Last,
+    #[serde(alias = "Full")]
+    Crate,
+    #[serde(alias = "Last")]
+    Module,
```

Comment:
Does `rename_all` apply to aliases as well? I would expect it to but I'm not entirely sure

Reply:
I don’t know, seems easy enough to check with playground? I’d expect that the answer is no — more flexible that way.

Reply:
ye it doesn't, you are also right this way its more flexible anyways